packages/mds-provider-processor/package.json

```diff
@@ -36,5 +36,7 @@
     "loglevel": "1.6.6",
     "uuid": "3.3.3"
   },
-  "devDependencies": {}
-}
\ No newline at end of file
+  "devDependencies": {
+    "husky": "^4.2.1"
+  }
```

Comment:
`devDependencies` should be in the root `package.json` ... `husky` is already installed.

Reply:
Not sure how this got there, will remove